modules/search-pipeline-common/src/yamlRestTest/resources/rest-api-spec/test/search_pipeline/10_basic.yml

```diff
@@ -12,5 +12,6 @@
       nodes.info: {}
 
   - contains:  { nodes.$cluster_manager.modules: { name: search-pipeline-common } }
-  - contains:  { nodes.$cluster_manager.search_pipelines.request_processors: { type: filter_query } }
-  - contains:  { nodes.$cluster_manager.search_pipelines.response_processors: { type: rename_field } }
+  - contains:  { nodes.$cluster_manager.search_pipelines.processors: { type: filter_query } }
+  - contains:  { nodes.$cluster_manager.search_pipelines.processors: { type: rename_field } }
+  - contains:  { nodes.$cluster_manager.search_pipelines.processors: { type: script } }
```

Comment:
I think this was a bad merge?

These now need to check for the appropriate type of processor, since they're now split by type.